src/main/java/org/json/XMLParserConfiguration.java

```diff
@@ -153,13 +156,14 @@ public XMLParserConfiguration (final boolean keepStrings, final String cDataTagN
      */
     private XMLParserConfiguration (final boolean keepStrings, final String cDataTagName,
             final boolean convertNilAttributeToNull, final Map<String, XMLXsiTypeConverter<?>> xsiTypeMap, final Set<String> forceList,
-            final int maxNestingDepth, final boolean closeEmptyTag) {
+            final int maxNestingDepth, final boolean closeEmptyTag, final boolean shouldTrimWhiteSpace) {
```

Comment:
Thanks for the feedback, this was done to ensure the clone() methods functions as required and the public constructor was left untouched. I have pushed a new commit to adjust this so that the constructor is untouched and the parameter is set in the clone method.